### ISSUE TITLE
sticker visibility option not available.

### ISSUE THREAD

**Issue by deepaksharma0406** [wuapnjie/StickerView#123]
sticker visibility option not available.

is it possible to maintain visibility a selected sticker not remove it totally.

**Comment by vishrut2207** [comment]
yes, that option not available in the current version. but you can achieve this by Opacity.
